activities/Paint.activity/lib/tutorial.js

```diff
@@ -0,0 +1,162 @@
+define(["webL10n"], function (l10n) {
+	var tutorial = {};
+
```

Comment:
Please use tab based indentation like the other files . in this file you have used space based indentation .